static/index.html

```diff
@@ -6,55 +6,122 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendago</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <!-- Bootstrap Font Icon CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <style>
+        
+    </style>
+   
 </head>
 <body>
 
-    <div class="container">
-        <h1>Generate a Custom Calendar</h1>
-        <form enctype="multipart/form-data" action="calendar" method="post">
-            <div class="field">
-                <label for="Year">Year</label>
-                <input type="number" name="Year" id="Year" placeholder="Year" />
+ 
+
+
+     <!-- contact form -->
+  <section id="contact  ">
+    <div class="container-xxl "  >
+        
+      <div class="text-center py-5">
+        <img src="../img/squre.jpg" width="200" class="rounded mx-auto d-block" alt="...">
+        <h2 class="display-5 fw-bold">Generate a Custom Calendar</h2>
+        <p class="display-6">Choose the presets for your calender...</p>
+      </div>
+   
+      <div class="row justify-content-center my-">
+        <div class="col-lg-6">
+          
+          <form>
```

Comment:
Change the form to use a post to the "/calendar" endpoint

`<form method="post" action="/calendar" enctype="multipart/form-data">`